docs/how_to/enabling_access_to_vmpooler.md

```diff
@@ -0,0 +1,18 @@
+# Enabling access to the vmpooler VMs during an acceptance test run
+
+If you are running acceptance tests for Beaker that utilise vmpooler, you will need to run an SSH agent and add the acceptance RSA key prior to running the tests
+
+To load the SSH agent and add the acceptance key, run the following:
+
+~~~bash
+eval `ssh-agent`
+ssh-add ~/.ssh/id_rsa-acceptance
+~~~
+  
+Now run acceptance tests that use the vmpooler:
+
+~~~bash
+bundle exec rake test:subcommands
```

Comment:
I guess thought it was okay to just use the rake tasks for running Beaker since the focus for this doc is enabling the SSH agent for Beaker runs, rather than focusing on how to run Beaker itself. In this case, the .ssh/id_rsa-acceptance key is specified via the `acceptance/config/acceptance-options.rb` file, but you could also specify the SSH key via the --keyfile flag